src/native-common/Image.tsx

```diff
@@ -91,6 +91,7 @@ export class Image extends React.Component<Types.ImageProps, Types.Stateless> im
                 onError={ this._onError }
                 shouldRasterizeIOS= { this.props.shouldRasterizeIOS }
                 { ...additionalProps }
+                tooltip = { this.props.title }
```

Comment:
Minor: Please move this above the expansion of additionalProps. Also, remove spaces around equals sign. (There's also an extra space after the equals for shouldRasterizeIOS — please remove it while you're in there.)